README.md

```diff
@@ -1,27 +1,32 @@
-# The Rust Programming Language
+<div align="center">
+  <img alt="The Rust Programming Language: A language empowering everyone to build reliable and efficient software"
+       src="https://www.rust-lang.org/static/images/rust-social-wide.jpg"
+       width="512" height="256">
```

Comment:
I think we can fix the dark mode thing as a followup maybe but I checked with a couple people and am getting "logo too big" feedback. It's especially annoying on smaller devices, please try the [srcset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#srcset) attribute as a fix?

Reply:
Alternatively twiddling the `width` as a % may be better?

Reply:
What about `width="50%"`?

![Capture d’écran 2024-04-01 à 21 38 15](https://github.com/rust-lang/rust/assets/3616612/fd64e51b-3542-4085-a78f-9cfbabb3a5ec)
![Screenshot 2024-04-01 at 21-38-32 Urgau_rust at improve-readme](https://github.com/rust-lang/rust/assets/3616612/9409d2c9-668e-4d23-a9a8-744ed25bbb2d)